main.go

```diff
@@ -388,6 +389,32 @@ func main() {
 		log.Donef("Using system installed CocoaPods version")
 	}
 
+	// Check ruby version
+	// Run this logic only in CI environment when the ruby was installed via rbenv for the virtual machine
+	if os.Getenv("CI") == "true" && rubycommand.RubyInstallType() == rubycommand.RbenvRuby {
+		fmt.Println()
+		log.Infof("Check selected ruby is installed")
+
+		rubyInstalled, rversion, err := rubycommand.IsSpecifiedRbenvRubyInstalled(configs.SourceRootPath)
+		if err != nil {
+			log.Errorf("Failed to check if selected ruby is installed, error: %s", err)
```

Comment:
Errors shouldn't be fail or warning instead?

Reply:
I think this should be an error and not a warning as there was an error during the command run. 
But I would not fail the step itself as the step's main purpose is to install the Cocoapods and not the ruby install itself.